### ISSUE TITLE
Add to DefinitelyTyped

### ISSUE THREAD

**Issue by psnider** [yankee42/typescript-server-sent-events#2]
Have you submitted this to DefinitelyTyped?

This file looks good to me.
It just needs a test file.
I'll help if you like...

**Comment by yankee42** [comment]
No, what do I need to do for that?
Feel free to create a pull request with the test file.

**Comment by maxime1992** [comment]
I think it would be a great idea to submit indeed !
The only definition I could find for SSE is yours ... :)
https://microsoft.github.io/TypeSearch/

And I've never published a type, but I think you just need to make a PR here : https://github.com/DefinitelyTyped/DefinitelyTyped It's explained in the readme.

**Comment by adumesny** [comment]
Yes it would be great as you are the only SSE typedef I could find as well... 

surprised SSE is so hidden (I know IE is the only browser NOT supporting it, maybe why MSFT doesn't include in their DOM definition either (can't do 'if (window.EventSource) {...use EventSource} else { use long polling/polyfill}

I have made a few more tweaks as I'm starting to use it, which I can push back, like:

interface IOnMessageEvent extends Event {
        data: string;
        origin: string;
    }
    interface IOnErrorEvent extends Event {
        target: IEventSourceStatic;
    }
